documentation/bolt_command_reference.md.erb

```diff
@@ -0,0 +1,18 @@
+# Bolt command reference
+
+These subcommands, actions, and options are available for Bolt.
+
+
+<% for command in @commands %>
+## <%= command[0] %>
+
+```plain
+<%= command[1][:banner] -%>
+```
+
+| Option | Description |
+| ------ | ----------- |
+<% for option in command[1][:options] -%>
+| <% if option[:short] %>`<%= option[:short] %>`, <% end -%>`<%= option[:long] -%><%= option[:arg] %>` | <%= option[:desc] %> |
+<% end %>
+<% end %>
```

Comment:
```suggestion
# Bolt command reference

These subcommands, actions, and options are available for Bolt.


<% @commands.each do |key, val| %>
## <%= key %>

\```plain
<%= val[:banner] -%>
\```

| Option | Description |
| ------ | ----------- |
<% val[:options].each do |option|  -%>
| <% if option[:short] %>`<%= option[:short] %>`, <% end -%>`<%= option[:long] -%><%= option[:arg] %>` | <%= option[:desc] %> |
<% end %>
<% end %>
```

Reply:
The iteration in the templates using indexing does not feel very ruby-like to me. I think you can use something like the suggestion.

Reply:
Updated. We should probably update `reference.md.erb` to be more ruby-like as well.